AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Commands/AWSSDKSwiftCLI/Subcommands/PrepareRelease.swift

```diff
@@ -165,6 +165,14 @@ struct PrepareRelease {
         do {
             try "\(newVersion)".write(toFile: "Package.version" , atomically: true, encoding: .utf8)
             log("Updated Package.version: \(newVersion)")
+            if (repoType == .awsSdkSwift) {
+                try "\(newVersion)".write(
+                    toFile: "Sources/Core/AWSClientRuntime/Sources/AWSClientRuntime/Resources/sdk-version.txt",
+                    atomically: true,
+                    encoding: .utf8
+                )
+                log("Updated AWSClientRuntime/Resources/Package.version: \(newVersion)")
+            }
```

Comment:
**Non-Blocking:** This may be the best solution but it gives me pause that we will be storing a duplicate of `package.version` in a `sdk-version.txt`. I understand the reasons for doing so but I wonder if there is either a better way or an opportunity for refactoring so that we only have a single file with the version.